pkgs/development/compilers/wren/default.nix

```diff
@@ -0,0 +1,45 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, python3
+, callPackage
+}:
+let
+  cli =
+    callPackage ./cli.nix { };
+in
+stdenv.mkDerivation rec {
+  pname = "wren";
+  version = "0.4.0";
+
+  src = fetchFromGitHub {
+    owner = "wren-lang";
+    repo = pname;
+    rev = version;
+    sha256 = "0w8n5lyn3wa1nmdyci0zi249w1qbq725cr1d9xsg067irq17v8k5";
```

Comment:
Prefer `hash = ...` to `sha256 = ...` Both currently work, but the former will make it easier if the algorithm ever needs to change.

Since there could be different hashing algorithms in use, this make sense.